projects/common/01bsp_radio_tx/01bsp_radio_tx.c

```diff
@@ -21,7 +21,7 @@ remainder of the packet contains an incrementing bytes.
 
 //=========================== defines =========================================
 
-#define LENGTH_PACKET   125+LENGTH_CRC // maximum length is 127 bytes
+#define LENGTH_PACKET   30+LENGTH_CRC // maximum length is 127 bytes

```

Comment:
I suggest to use the max length in this little test, as things always break in the corner cases. I understand that the PDU of IEEE802.15.4g is >>127 bytes, but since this application in used on all boards, we should leave it to 127B.